projects/EIP-x_stateless_verification_with_LC_which_consume_ZKP_Portal.md

```diff
@@ -0,0 +1,71 @@
+
+##EIP-x 
+ Stateless LC That Consumes ZKP With Efficient Access To The Specific Segments Of The State.
+
+
+## Motivation
+In the dynamic landscape of blockchain technology, addressing the limitations of traditional light clients is paramount for the widespread adoption and efficient functioning of the Ethereum network. Conventionally, light clients have burdened full nodes by incrementally requesting block state, posing challenges to network scalability. The integration of Verkle trees has provided a significant step forward, facilitating smoother transitions for lightweight clients between blocks. However, a lingering concern lies in the assurance of accurate state root confirmation, highlighting the need for further advancements. The inability of light clients to handle zero-knowledge proofs compounds the complexity. Moreover, portal nodes, a crucial bridge in this ecosystem, face the challenge of effectively prioritizing essential information for streamlined and swift stateless light client operations. Overcoming these obstacles is central to empowering light clients and optimizing the Ethereum network for a seamless and inclusive decentralized future.
+
+# Project Description
+Project Description:
+
+Our endeavor,EIP-x, embodies a thoughtful exploration of Helios light clients' capabilities to transform the way we access and validate blockchain data. At its core, the State Provider entity operates as a conduit to access the most recent block state of a block in a manner that fosters trust without reliance on intermediaries. This process entails the extraction of a cryptographic witness, which in turn serves as the foundation for the creation of zero-knowledge proofs (ZKPs). These ZKPs, integral to our approach, are subsequently disseminated to all participating lightweight client (LC) nodes across the network.
```

Comment:
I am getting a bit confused about your work here. Are you working on Helious, Trin or flashbot tech? It's missing context of your eip proposal

Reply:
thanks, Mario for pointing this out, We provide two entities under one project: EIP-x,  there are two types of nodes, one is executor (trin) and helios is a provider node. One entity will extract the last state root hash and generate Witness and a ZKP the other consuems the zko and can verify the proof, ultimately verify the Tx in stateless manner